vars/sbtReleaseDeployJob.groovy

```diff
@@ -27,7 +22,7 @@ def call(Map config) {
         agent {
           ecs {
             inheritFrom "base"
-            taskDefinitionOverride "arn:aws:ecs:eu-west-2:${env.MANAGEMENT_ACCOUNT}:task-definition/s3publish-${config.stage}:2"
+            taskDefinitionOverride "arn:aws:ecs:eu-west-2:${env.MANAGEMENT_ACCOUNT}:task-definition/s3publish:2"
```

Comment:
Assume this works for all jobs calling this?

Reply:
Yep. I'm going to change the jobs calling it so they don't pass the stage parameter because it's not needed but they'll work as is with this change.